src/components/Breadcrumb/style.scss

```diff
@@ -0,0 +1,41 @@
+@import "styles/variables";
+@import "styles/mixins";
+
+.breadcrumb {
+    padding: 12px 24px;
+    background: $grey-bg;
+    ol {
+        display: flex;
+        > li {
+            display: inline-flex;
+            align-items: center;
+            padding: 0;
+            a {
+                display: block;
+                padding: 12px;
+                color: $gray-80;
+                font-size: 12px;
```

Comment:
rather than defining the fonts explicitly for each element, let's find the font in the design system that matches, 
 
so this looks to me like it's the "OVERLINE" font here https://www.figma.com/file/2t8Wcyu8FAXxzQGpyK8r4f/TC-Design-System-2.0-%2F-2022?node-id=2%3A361
 
we would just need to override the font-weight